Mathlib/Data/Quot.lean

```diff
@@ -833,3 +833,15 @@ instance (q₁ : Quotient s₁) (q₂ : Quotient s₂) (f : α → β → Prop)
   Quotient.lift₂.decidablePred _ h _ _
 
 end Quotient
+
+lemma Equivalence.quot_mk_eq_iff {α : Type _} {r : α → α → Prop} (h : Equivalence r) (x y : α) :
```

Comment:
Should this be a simp lemma?

Reply:
Maybe. I will try to see if it would break anything...

Reply:
He does not break anything!